doc/developing.md

```diff
@@ -97,4 +97,23 @@ When updating the supported GHC versions, remember to update:
 - [The README](../README.md)
 - [The Cabal file](../llvm-pretty-bc-parser.cabal)'s `Tested-with` field
 - [The Nix flake](../flake.nix)
+
+  Run `nix flake update` (installing [`nix`](https://nixos.org) first if
+  necessary) which will update the `flake.lock` file in the top level directory.
+  Optionally run `nix build ./#TESTS` to run the nix-based tests locally.  Then
+  commit the updated `flake.lock` file.
```

Comment:
My understanding is that `nix flake update` will update _everything_ in the lock file to the latest version, is that correct? What would be the workflow if you only wanted to update a single dependency (e.g., the `llvm-pretty` submodule)?